cli/src/cluster_query.rs

```diff
@@ -1342,6 +1359,32 @@ pub fn process_get_transaction_count(rpc_client: &RpcClient, _config: &CliConfig
     Ok(transaction_count.to_string())
 }
 
+// if `--additional-fee` is not specified, no comput-budget instruction will be included in
+// transaction;
+// otherwise, user specified additional-fee lamports will be used for compute-budget instruction;
+// if `0` is specified, the max-fee of one signature will be used as default.
+fn build_compute_budget_instruction(additional_fee: &Option<u32>) -> Option<Instruction> {
+    let mut additional_fee = (*additional_fee)?;
```

Comment:
```suggestion
fn build_compute_budget_instruction(additional_fee: u32) -> Instruction {
```
I do love using try `?` on Options, but I think we should probably just make this infallible, and bump the Option inspection up a level (see previous comment/suggestion)